client/src/components/generic/Table.js

```diff
@@ -92,8 +92,8 @@ export const Table = ({ order, orderBy, onRequestSort, columns, rows, isLoading,
             </StyledTableRow>
           )) : pageRows.map((row, index) => (
             <StyledTableRow hover key={index}>
-              {Object.keys(row).map((key) => (
-                <StyledTableCell key={key}>{row[key]}</StyledTableCell>
+              {columns.map((column) => (
+                <StyledTableCell key={column.id}>{row[column.id] || ''}</StyledTableCell>
```

Comment:
@mauriciofreshw I'm not sure where you added the logic for empty cells, but this should handle it now. So we can send an array of any object to a table and the props should align with columns as well as empty cells being handled correctly.

Reply:
The value when null was being set as '', but the way you did here is better.

Reply:
Yeah, I figured we'd be doing the same thing everywhere that Table is used. This will save us from repeating ourselves.